test/test.js

```diff
@@ -51,7 +51,7 @@ test('markdown', (t) => {
     t.equal(result.source, fixture, 'filename');
     t.deepEqual(_.orderBy(result.warnings, ['line', 'column']), markdownExpectedWarnings);
     t.end();
-  }).catch(t.end);
+  }).catch(t.threw);
```

Comment:
`t.end` in catch will overlook error in promise, even though source code had error, test code would passed. look [tap promises example](http://www.node-tap.org/promises/).